.github/workflows/publish.yaml

```diff
@@ -5,6 +5,7 @@ on:
 jobs:
   publish:
     runs-on: ubuntu-latest
+    timeout-minutes: 5
```

Comment:
why is timeout 5 minutes vs 120 minutes [here](https://github.com/canonical/xtrabackup-snap/pull/4/files#diff-e81e13daadd1745de51d8b8d85fce1fcd9be355732b64d60a6b56e18c28388caR11)?

Reply:
here we just install APT packages, and there we are building from sources.